{{cookiecutter.project_shortname}}/code/setup.cfg

```diff
@@ -0,0 +1,40 @@
+
+[metadata]
+name = {{cookiecutter.project_shortname}}
+version = attr: {{cookiecutter.project_package}}.__version__
+description = Invenio Research Data Management.
+long_description = file: README.rst, CHANGES.rst
+license = MIT
+author = {{cookiecutter.author_name}}
+author_email = {{cookiecutter.author_email}}
+platforms = any
+url = {{cookiecutter.github_repo}}
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=cookiecutter.python_version.split()[0]
+zip_safe = False
+
+[options.extras_require]
+tests =
+    pytest-black>=0.3.0
+    pytest-invenio>=2.1.0,<3.0.0
```

Comment:
minor: why the upper cap only in one?

Reply:
I copied it from invenio-app-rdm. That said, I don't know the reason why we cap only the one...Will check!